<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorConfig</class>
 <widget class="QWidget" name="SensorConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>传感器配置</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>721</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    gridline-color: #d0d0d0;
    border: 1px solid #999999;
    selection-background-color: #66aaff;
    selection-color: white;
    alternate-background-color: #f0f0f5;
}

QTableView::item {
    padding: 5px;
    font-size: 12px;
    color: #333333;
}

QTableView::item:selected {
    background-color: #66aaff;
    color: white;
}

QTableView::horizontalHeader {
    background-color: #e0e0e0;
    border-bottom: 1px solid #999999;
}

QTableView::horizontalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-right: 1px solid #d0d0d0;
}

QTableView::verticalHeader {
    background-color: #e0e0e0;
    border-right: 1px solid #999999;
}

QTableView::verticalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-bottom: 1px solid #d0d0d0;
}</string>
   </property>
  </widget>
  <widget class="QTableView" name="TD_tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>151</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    background-color: #ffffff;
    gridline-color: #d0d0d0;
    border: 1px solid #999999;
    selection-background-color: #66aaff;
    selection-color: white;
    alternate-background-color: #f0f0f5;
}

QTableView::item {
    padding: 5px;
    font-size: 12px;
    color: #333333;
}

QTableView::item:selected {
    background-color: #66aaff;
    color: white;
}

QTableView::horizontalHeader {
    background-color: #e0e0e0;
    border-bottom: 1px solid #999999;
}

QTableView::horizontalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-right: 1px solid #d0d0d0;
}

QTableView::verticalHeader {
    background-color: #e0e0e0;
    border-right: 1px solid #999999;
}

QTableView::verticalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-bottom: 1px solid #d0d0d0;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tietlel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#00aaff;&quot;&gt;传感器配置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>581</width>
     <height>93</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 8em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 8em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 8em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TishiButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 8em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="bg_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>901</width>
     <height>601</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>901</width>
     <height>601</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>bg_label</zorder>
  <zorder>tableView</zorder>
  <zorder>TD_tableView</zorder>
  <zorder>Tietlel</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
